src/components/Slider/SliderBase/SliderRail.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { forwardRef } from "react";
+import PropTypes from "prop-types";
+import { bem } from "../SliderCommons";
+
+const SliderRail = forwardRef(({ className, children, onClick }, ref) => {
+  function handleClick(e) {
```

Comment:
I think create function inside the render function will create new instance for each render of the slider rail. Why not use useCallback ?